frontend/src/js/common-ui/forms/__snapshots__/TimeFramePicker.test.tsx.snap

```diff
@@ -360,83 +472,337 @@ label[data-shrink=false]+.MuiInputBase-formControl .emotion-3:focus::-ms-input-p
+              inputmode="numeric"
+              role="spinbutton"
+              spellcheck="false"
+              tabindex="-1"
+            >
+              MM
+            </span>
+            <span
+              class="MuiPickersInputBase-sectionAfter emotion-5"
+            >
+              -
+            </span>
+          </span>
+          <span
+            class="MuiPickersSectionList-section emotion-4"
+            data-sectionindex="2"
+          >
+            <span
+              class="MuiPickersInputBase-sectionBefore emotion-5"
+            />
+            <span
+              aria-disabled="false"
+              aria-label="Day"
+              aria-labelledby=":r1:-day"
+              aria-readonly="false"
+              aria-valuemax="31"
+              aria-valuemin="1"
+              aria-valuetext="Empty"
+              autocapitalize="off"
+              autocorrect="off"
+              class="MuiPickersSectionList-sectionContent MuiPickersInputBase-sectionContent emotion-6"
+              contenteditable="true"
+              id=":r1:-day"
+              inputmode="numeric"
+              role="spinbutton"
+              spellcheck="false"
+              tabindex="-1"
+            >
+              DD
+            </span>
+            <span
+              class="MuiPickersInputBase-sectionAfter emotion-5"
+            />
+          </span>
+        </div>
+        <div
+          class="MuiInputAdornment-root MuiInputAdornment-positionEnd MuiInputAdornment-outlined MuiInputAdornment-sizeMedium emotion-16"
+        >
+          <button
+            aria-label="Choose date"
+            class="MuiButtonBase-root MuiIconButton-root MuiIconButton-edgeEnd MuiIconButton-sizeMedium emotion-17"
+            tabindex="0"
+            type="button"
+          >
+            <svg
+              aria-hidden="true"
+              class="MuiSvgIcon-root MuiSvgIcon-fontSizeMedium emotion-18"
+              data-testid="CalendarIcon"
+              focusable="false"
+              viewBox="0 0 24 24"
+            >
+              <path
+                d="M17 12h-5v5h5v-5zM16 1v2H8V1H6v2H5c-1.11 0-1.99.9-1.99 2L3 19c0 1.1.89 2 2 2h14c1.1 0 2-.9 2-2V5c0-1.1-.9-2-2-2h-1V1h-2zm3 18H5V8h14v11z"
+              />
+            </svg>
+          </button>
+        </div>
         <fieldset
           aria-hidden="true"
-          class="MuiOutlinedInput-notchedOutline emotion-4"
+          class="MuiPickersOutlinedInput-notchedOutline emotion-19"
         >
           <legend
-            class="emotion-5"
+            class="emotion-20"
           >
-            <span>
+            <span
+              class="emotion-21"
+            >
               From
             </span>
           </legend>
         </fieldset>
+        <input
+          aria-hidden="true"
+          aria-label="From"
+          class="MuiPickersInputBase-input MuiPickersOutlinedInput-input emotion-22"
+          id=":r2:"
+          tabindex="-1"
+          value=""
```

Comment:
just a heads up @mineralsfree as the date pickers are now quite a step from a regular input (considering their `span` group representation above), making the access to the input value a bit clunky IMO